pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -100,6 +100,23 @@ type VSphereConfig struct {
 		VMName string `gcfg:"vm-name"`
 	}
 
+	VirtualCenter map[string]*struct {
+		// vCenter username.
+		User string `gcfg:"user"`
+		// vCenter password in clear text.
+		Password string `gcfg:"password"`
+		// vCenter port.
+		VCenterPort string `gcfg:"port"`
+		// True if vCenter uses self-signed cert.
+		InsecureFlag bool `gcfg:"insecure-flag"`
+		// Datacenter in which VMs are located.
+		Datacenter string `gcfg:"datacenter"`
+		// WorkingDir is path where VMs can be found.
+		WorkingDir string `gcfg:"working-dir"`
```

Comment:
I think user may want us to use different workingDir on different VCs.
If on all VCs workingDir is same, then user can specify only once in Global.